Cython/TestUtils.py

```diff
@@ -253,4 +275,4 @@ def write_newer_file(file_path, newer_than, content, dedent=False):
         other_time = None
 
     while other_time is None or other_time >= os.path.getmtime(file_path):
-        write_file(file_path, content, dedent=dedent)
+        os.utime(file_path)
```

Comment:
If the only thing we need is to change the timestamp, then it makes little sense to load the system with I/O operations with potentially large amounts of data.

Reply:
This is used with literals strings in test functions. I don't expect those "large amounts of data" to reach even a kilobyte.

It seemed simpler as it was, since it only did one thing, twice, that was easy to understand, but I don't care enough to fight for it.

Reply:
(Also, just for the record: in tests, this file will almost certainly be in a temp folder. Any decent operating system would only create it in memory and never even write it to disk.)

Reply:
> I don't expect those "large amounts of data"


> potentially

Reply:
> in tests, this file will almost certainly be in a temp folder. Any decent operating system would only create it in memory and never even write it to disk

Also true

Reply:
> I don't care enough to fight for it.

We are not fighting, but looking for a middle ground. Aren't we?